scripts/docker_run

```diff
@@ -3,11 +3,19 @@
 set -o errexit
 set -o xtrace
 
-docker build --tag=oak .
+USERNAME=$(id -un)
+
+mkdir -p $PWD/bazel-cache
```

Comment:
I think we already use `.` instead of `$PWD` in other places, so we should stick to that?

Reply:
We need to keep $PWD for docker volumes, as it doesn't work with .